tests/integration/views/test_session.py

```diff
@@ -52,3 +52,24 @@ def test_expires_at_future_time(self, mock_redis):
         # Then the expiration time is in 40 minutes as calling expires-at doesn't extend the session
         self.assertEqual(expected_expires_at, parsed_json["expires_at"])
         self.assertEqual((future_time + timedelta(minutes=40)).isoformat(), expected_expires_at)
+
+    @freeze_time(TIME_TO_FREEZE)
+    @patch("redis.StrictRedis.get")
+    def test_patch_expires_at_current_time(self, mock_redis):
```

Comment:
This seems a bit complicated, I think all we need to do is check that the patch extends the timeout. Set a time using freeze, walk it forward 20 minutes (or whatever) call patch and make sure the session timeout is 60 minutes further along.

Reply:
I've made it simpler now.